crates/store/src/db/schema.rs

```diff
@@ -50,8 +50,18 @@ diesel::table! {
         account_id -> Binary,
         network_account_id_prefix -> Nullable<BigInt>,
         account_commitment -> Binary,
+        code_commitment -> Binary, // double check if this can ever be the case, I don't think so
+        storage -> Nullable<Binary>,
+        vault -> Nullable<Binary>,
+        nonce -> Nullable<BigInt>,
         block_num -> BigInt,
-        details -> Nullable<Binary>,
+    }
+}
+
+diesel::table! {
+    account_codes (code_commitment) {
+        code_commitment -> Binary,
+        code -> Binary,
```

Comment:
More of a philosophical question, but why do we call it account code, and then note script, transaction script?